src/m2-localize.asm

```diff
@@ -1,10 +1,13 @@
+//Uncompressed Graphics
 .org 0x876c1fc :: .incbin "data/m2-door-bubble.bin" //Door - Don->!
 .org 0x8787fbc :: .incbin "data/m2-nusutto-sign.bin" //Nusutto->Burglin Park
 .org 0x869ff28 :: .incbin "data/m2-ness-pajamas.bin" //Ness' Pajamas
 .org 0x874d4bc :: .incbin "data/m2-runaway-five.bin" //Runaway Five Sprites
 
-.org 0x8CC6D30
+//Compressed Graphics Data
+.org 0x8BA5630
 m2InsaneCultist:
```

Comment:
Since it looks like this is starting mostly clean, it would probably be a good idea to divide stuff up now (i.e. one file for in-place incbins and pointers, one for repointed data). The arbitrary offsets aren't needed that way, because you can include the repointed data one later in m2-hack.asm.

Reply:
@Sterophonick were you going to consider these changes in this PR or a later one? Just wondering if we should merge yet